### ISSUE TITLE
[Bug Report] Batch Scraped performer Country data from stash-box is only two-lettered

### ISSUE THREAD

**Issue by grimlertk** [stashapp/stash#1567]
**Describe the bug**
When the data for performers is scraped from stashbox, it sets country names with only two letters (US,CO,CA, etc.) instead of United States, Colombia, Canada, and so on. This results in stash not recognizing the country name of the performer.

**To Reproduce**
1. Go to Performers tab
2. Click on Tagger button
3. Click on Batch Add Performers button
4. Enter performer names and click on Add Performers
4. The scraped data for the new performers will contain the Country names in short forms, which messes up stash's country recognition

**Expected behavior**
The country names fetched should be expanded, not in their short forms

**Stash Version: (from Settings -> About):** v0.8.0

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Browser: Microsoft Edge (Chromium)
 - Version: 91.0.864.64 (Official build) (64-bit)
 
**Additional context**
If the data for performers is scraped from their individual page, then the country names are added properly.

**Comment by DampToast** [comment]
I looked into this a bit and the graphql call to scrape the performer in stashdb returns the 2 character country code.  I also checked StashDB and it's sending the 2 character code to the front end as well.  If we want to make StashApp convert all 2 character results from country as 2 character ISO codes, I can write something up to get them converted possibly.  It's already being done in Stash Box so I'm sure I can just borrow that code and tweak it if need be, then just find where I need to call it in StashApp.

https://github.com/stashapp/stash-box/blob/8bcf504f8c844342d63df5a48b5b254195ba392e/frontend/src/utils/country.ts
